webpack/components/extensions/RegistrationCommands/fields/ActivationKeys.js

```diff
@@ -88,7 +94,7 @@ ActivationKeys.propTypes = {
 };
 
 ActivationKeys.defaultProps = {
-  activationKeys: undefined,
+  activationKeys: [],
```

Comment:
Note to myself: Why it was `undefined` in the first place?

Reply:
Activation key is set to "[]" in the useEffect of indexJs

Reply:
yeah I know, but it was supposed to be `[]` from the beginning ...